distribution/packages/build.gradle

```diff
@@ -407,6 +407,9 @@ subprojects {
     'default' buildDist
   }
 
+  if (! (dpkgExists() || rpmExists())) {
+    return;
```

Comment:
Let's avoid short-circuiting configuration closures with `return` statements. It can be easy to miss. Let's just nest the associated config in the if statement.

Reply:
Also, can we write this in our style as:

```
if (dpkgExists() == false && rpmExists() == false) {
  return;
}
```

As you know, like it or not we've all come to accept it, we prefer the `== false` over the use of the unary not, and I think it reads more clearly this way: if `dpkg` does not exist and `rpm` does not exist...

Reply:
Thanks, I got quite used to it myself, was surprised this slipped trough